people-and-planet-ai/land-cover-classification/visualize.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import numpy as np
+import plotly.graph_objects as go
```

Comment:
I think `numpy as np` is pretty common but I have not seen `plotly."g"raph_"o"bject` being referred to as `go`; although it makes sense after knowing it, consider using `from plotly import graph_objects` instead unless `go ~== graph_object` is a common industry abbr.